js/script.js

```diff
@@ -89,3 +73,48 @@ $(document).ready(function () {
     });
 });
 
+// 30 second timer
+let timer;
+let timeLeft = 30; // seconds
+
+let zombieAppear;
+
+/**
+ * Start Game Function
+ */
+ function startGame() {
+    resetGame();
+    timer = setInterval(updateTimer, 1000);
+    zombieAppear = setInterval(popupZombie, 1500);
+}
+
+// decrements time left by one if time left is greater than zero.
```

Comment:
Inconsistent function comments. Some have `/**...*/` and some just have `//`.
Also, some have a blank line before the function signature, some don't.